packages/manager/modules/pci/src/projects/project/additional-ips/additional-ips.html

```diff
@@ -0,0 +1,135 @@
+            data-on-go-to-region="$ctrl.goToRegion()"
+        ></pci-announcement-banner>
+
+        <cui-message-container
+            data-messages="$ctrl.messages"
+        ></cui-message-container>
+
+        <oui-tabs>
+            <oui-tabs-item
+                heading="{{:: 'pci_additional_ips_failover_ip_title' | translate}}"
+            >
+                <oui-datagrid
+                    data-rows="$ctrl.additionalIps"
+                    criteria="$ctrl.criteria"
+                    on-criteria-change="$ctrl.onListParamChange($criteria)"
+                >
+                    <oui-datagrid-topbar>
+                        <oui-action-menu
+                            data-text="{{:: 'pci_additional_ips_actions' | translate }}"
+                        >
+                            <oui-action-menu-item
+                                state="pci.projects.project.additional-ips.imports"
+                            >
+                                <span
+                                    data-translate="pci_additional_ips_import_failover_ip"
+                                ></span>
+                            </oui-action-menu-item>
+                            <oui-action-menu-item
+                                data-on-click="$ctrl.goToAdditionalIpOrderPage()"
+                            >
+                                <span
+                                    data-translate="pci_additional_ips_order"
+                                ></span>
+                            </oui-action-menu-item>
+                        </oui-action-menu>
+                    </oui-datagrid-topbar>
+                    <oui-datagrid-column
+                        data-title=":: 'pci_additional_ips_failover_ip_title' | translate"
+                        data-property="ip"
+                        data-type="string"
+                        data-searchable
+                        data-filterable
+                    >
+                    </oui-datagrid-column>
+                    <oui-datagrid-column
+                        data-title=":: 'pci_additional_ips_ip_block' | translate"
+                        data-property="block"
+                        data-type="string"
+                        data-searchable
+                        data-filterable
+                    >
+                    </oui-datagrid-column>
+                    <oui-datagrid-column
+                        data-title=":: 'pci_additional_ips_geoloc' | translate"
+                        data-property="geoloc"
+                        data-type="string"
+                    >
+                        <span
+                            data-ng-bind="'pci_additional_ips_country_' + $row.geoloc | translate"
+                        ></span>
+                    </oui-datagrid-column>
+                    <oui-datagrid-column
+                        data-title=":: 'pci_additional_ips_id' | translate"
+                        data-property="id"
```

Comment:
Related to this :
https://github.com/ovh/manager/pull/7290/files?#r891014894

Reply:
Id once and Ip once